test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -33,7 +33,7 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
 
   // A Wallet with some balance on Testnet.
   let wallet: Wallet
-  before(async function () {
+  beforeEach(async function () {
```

Comment:
`before` is called once before the entire test suite is run. this means that the `wallet` object in every test was actually the same, making the tests have side-effects on each other. 

this was causing my new test to fail, because a trust line were previously created from. this sort of sucks because it's slower but it makes the tests easier to reason about

Reply:
yeah... my `sendIssuedCurrency` integration tests are super wallet-heavy and I end up creating new wallets in each one to avoid side effects as well.  However, maybe we can figure out which tests don't cause side effects (or aren't affected by previous side effects) and use the same wallet across all of those as established in a `before`, and otherwise establish wallets in each test on an as-needed basis, instead of calling `beforeEach` for EVERY single test.. which may slow things down...

Reply:
Any chance you can put this back to `before` and create your new wallets inside your tests specifically?